EIPS/eip-3102.md

```diff
@@ -0,0 +1,189 @@
+---
+eip: 3102
+title: Binary trie structure
+author: Guillaume Ballet (@gballet), Vitalik Buterin (@vbuterin)
+status: Draft
+discussions-to: https://ethresear.ch/t/binary-trie-format/7621
+status: Draft
+type: Standards Track
+category: Core
+created: 2020-09-01
+---
+
+## Simple Summary
+
+A proposal for changing the format of the data-storage structure from hexary to binary, merging the account and storage tries, and using the faster BLAKE2. The goal is to make stateless proofs more wieldy.
+
+## Abstract
+
+This proposal presents a binary structure and merkelization rule for the account and storage tries, which are merged into a single “state” trie. RLP and most of the MPT’s optimizations are dropped to simplify the design. Keccak256 is replaced with BLAKE2-256.
```

Comment:
The Abstract should be a very terse human readable version of the specification.  Ideally, someone should be able to read the abstract and get a rough idea of what it would take to implement this specification.  Minutia and edge cases should be left out, but the core should be included.  In this case, this specification is pretty big so it will be hard to cut it down but the ideal abstract is 1 or two paragraphs in mostly plain English.